dev/releases/HelpLinks-to-JSON.g

```diff
@@ -0,0 +1,53 @@
+##  
+##  A utility for dumping links to all available manual sections 
+##  (for use with named manual references on web pages or in search
+##  utilities).
+##
+##  This is meant to be used with a plain vanilla GAP release, with
+##  only the bundled packages and no special user config.
+##
+
+LoadPackage("json");
+
+InstallMethod(_GapToJsonStreamInternal,
+[IsOutputStream, IsObject],
+function(o, x)
```

Comment:
What is this about? Perhaps add a comment?

Reply:
To be honest I don't know, and I will just delete this. For context, some of the code in this file was copied from `dev/releases.PackageInfos-to-JSON.g`. I copied this particular bit from the top of that file; it was added when the file was created in https://github.com/gap-system/gap/commit/8e158d6252d396a7dc04a70db42e7412e616232a#diff-526d642e5b9cfac653edfcfa495f06aa6a1f4f96dea9b26e352fa6f75a221252.

I guess that @ssiccha added that because a PackageInfo record could in theory contain arbitrarily weird GAP objects, and he wanted to write the script in a way that doesn't break when it encounters a crazy PackageInfo record. Therefore he installed this default method for `IsObject` (i.e. to catch unexpected things) which overwrote the JSON method.

But here, we _construct_ the record piece by piece that we feed to the JSON package; it is a record of records of strings. Therefore it's known to be safe, and we don't need such a custom fallback.